src/collections/_documentation/workflow/releases/health.md

```diff
@@ -41,6 +41,26 @@ The release details page focuses on an individual release. Elements of the relea
 
 [{% asset releases/release_details.png alt="View of the release details page showing an overview graph, files changed, and other details." %}]({% asset releases/release_details.png @path %})
 
+Hover over specific dates and times in the overview graph for a quick perspective of events that crashed, behaved abnormally, errored, or are healthy.
+
+Crashed 
+
+: The app had an explicit unhandled error or hard crash. You'll typically be able to see the corresponding issue in Sentry that captures this event. 
```

Comment:
@marandaneto @jan-auer How about this? Might be a good compromise for now?

Reply:
sounds good to me

Reply:
Sounds good, thanks!